### ISSUE TITLE
Other players' breaking progress will be reset when a shared tool loses durability

### ISSUE THREAD

**Issue by EnchantedHay** [Matistan/MinecraftSharedInventory#5]
Other players' breaking process will be reset when a shared tool loses durability.

**Comment by Matistan** [comment]
Yes, because when a player mines a block, a tool's durability will change, which will update this item for every player, and there is no fix for that (I think) .
